app/assets/stylesheets/manageiq-v2v/type-ahead-select.scss

```diff
@@ -0,0 +1,61 @@
+/*
+  Until we can properly import patternfly-react CSS extensions
+  (which will probably require changes to manageiq-ui-classic's webpack config)
+  we need to include the styles for react-bootstrap-typeahead manually.
+
+  This can probably be removed after https://github.com/patternfly/patternfly-react/issues/673 is fixed.
+  This is a copy of https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/sass/patternfly-react/_type-ahead-select.scss
+*/
+
+@import 'react-bootstrap-typeahead/css/Typeahead';
+
+.rbt-input-main {
+  padding: 2px 6px;
+}
+
+.rbt-input-multi {
+  height: fit-content;
```

Comment:
This is problematic.


According to https://developer.mozilla.org/en-US/docs/Web/CSS/height ,
`fit-content` is only supported in chrome.

Remember we need to support IE11.

Reply:
Thanks for noticing that. This CSS came directly from patternfly-react, and it appears to me that we don't need that style rule anyway. It supposedly was added to fix a bug that I can't seem to reproduce without it. I opened the issue https://github.com/patternfly/patternfly-react/issues/1362 to track that, and I'll remove that CSS from our copy for now. If it turns out to be a real issue, I figure we can track that separately from this PR.

Reply:
As long as this doesn't *break* anything, it's OK to be sub-optimal in IE :+1:

Reply:
Yeah, it seems like even if this issue is real (I'm having trouble reproducing it), the worst that would happen is that if a user tried to select more than something like 8-10 hosts, the overflow wouldn't be visible. I can't imagine users would need that many hosts anyway.

Reply:
I found an IE11-compatible fix for the upstream issue (https://github.com/patternfly/patternfly-react/pull/1369), so I'll go ahead and apply that fix to the copied styles here!